tests/OSPSuite.R.Tests/Services/DataImporterTaskSpecs.cs

```diff
@@ -0,0 +1,39 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using System;
+using System.IO;
+
+namespace OSPSuite.R.Services
+{
+   public abstract class concern_for_DataImporter : ContextForIntegration<IDataImporterTask>
+   {
+      protected override void Context()
+      {
+         base.Context();
+         sut = Api.GetDataImporterTask();
+      }
+
+      protected string getFileFulName(string fileName) => Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Data", fileName);
```

Comment:
typo: `getFileFulName` (sorry, didn't see it last time)

Reply:
Well, better later than never, right? ;) I did not see it before either and I wrote the code myself...